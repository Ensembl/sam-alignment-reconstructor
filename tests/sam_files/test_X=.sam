MS2007487-600V3:1:2109:08999:01136	163	cecr1a_T2_25_17051701-17051898	1	70	3=1X4=2X4=	=	1	197	CTTATATTGGCCTT	IIIIIIIIIIIIII	PG:Z:novoalign	AS:i:242	UQ:i:242	NM:i:16	MD:Z:3C4AT4	PQ:i:448	SM:i:70	AM:i:70	BC:Z:ATCCCG
