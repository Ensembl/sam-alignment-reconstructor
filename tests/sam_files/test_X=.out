MS2007487-600V3:1:2109:08999:01136
CTTCTATTATCCTT
|||:||||::||||
CTTATATTGGCCTT

